modules/twinkleblock.js

```diff
@@ -112,7 +110,6 @@ Twinkle.block.callback = function twinkleblockCallback() {
 			list: [{
 				checked: relevantUserName !== mw.config.get('wgRelevantUserName'), // In case the user closes and reopens the form
 				label: 'Block the /64 instead',
-				tooltip: 'Will eschew leaving a template.',
```

Comment:
Since this is new/weird, I think it'd be nice to note exactly what will happen.  It'll also be necessary when ranges are enabled, since in those cases we *will* skip the template.  Maybe something like:

```js
tooltip: Morebits.ip.isRange(mw.config.get('wgRelevantUserName')) ? 'Will eschew leaving a template.' : 'Any template issued will go to the original IP: ' + mw.config.get('wgRelevantUserName'),
```